.travis.yml

```diff
@@ -13,6 +13,7 @@ compiler:
 
 before_install:
   - if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+      sudo add-apt-repository ppa:as-bahanta/raqm -y;
```

Comment:
Please add a comment in the file why this PPA is needed, so we don’t miss this in the future.